user_interface.py

```diff
@@ -0,0 +1,158 @@
+import crud as cr
+import logger as lg
+
+
+print('\nВас приветствует телефонная книга')
+
+
+def ls_menu():
+    while True:
+        print('\nМЕНЮ')
+        print('1. Показать все записи.')
+        print('2. Найти номер по фамилии.')
+        print('3. Найти номер по имени.')
+        print('4. Поиск по номеру телефона.')
+        print('5. Добавить новую запись.')
+        print('6. Изменить существующую запись.')
+        print('7. Удалить запись.')
+        print('8. Закрыть программу.\n')
+        n = сhecking_the_number(input('Выберите пункт меню: '))
+
+        if n == 1:
+            lg.logging.info('The user has selected item number 1')
+            print(cr.retrive())
+
+        elif n == 2:
+            lg.logging.info('The user has selected item number 2')
+            search = input('Введите фамилию: ')
+            lg.logging.info('User entered: {search}')
```

Comment:
Вы забыли указать, что строка является форматируемой. Без этого у вас не будет подставляться туда значение переменной `search`.
```suggestion
            lg.logging.info(f'User entered: {search}')
```
![image](https://user-images.githubusercontent.com/49741194/209460014-b246502a-bd14-469e-afca-703cf764d2c9.png)